llvm/test/Transforms/InstCombine/fabs.ll

```diff
@@ -547,6 +547,20 @@ define double @select_fcmp_nnan_nsz_ult_zero_unary_fneg(double %x) {
   ret double %fabs
 }
 
+
+define float @absfloat32f_olt_fast_no_signed_zeroes(float %x) "no-signed-zeros-fp-math" {
```

Comment:
If there aren't already test cases showing that this won't happen without the function attribute, can you add them?

Reply:
Added tests with no attributes.